simpeg/electromagnetics/time_domain/receivers.py

```diff
@@ -120,6 +120,20 @@ def getTimeP(self, time_mesh, f):
         projected_time_grid = f._TLoc(self.projField)
         return time_mesh.get_interpolation_matrix(self.times, projected_time_grid)
 
+    def active_times(self, projection):
+        """Get active times for the receiver.
+
+        Parameters
+        ----------
+        projection : Sparse matrix
+
+        Returns
+        -------
+        numpy.ndarray
+            Active times for the receiver.
+        """
+        return np.unique(sp.find(projection)[1])
```

Comment:
I think we should be able to do since it is a sparse matrix 
```suggestion
        return projection.indices
```